Phat-Taupe-Pests/BangazonCLI
# Ben adding quantity to product

## REFERENCE

Ticket #6

[ERD](https://github.com/Phat-Taupe-Pests/Bangazon) 

## DESCRIPTION
Added quantity to the product. Updated model, manager, seeder, checker, tests, and all implementation code in the menu items that use product.
## STEPS TO TEST
1. Clone this branch to a local repo
2. Run `dotnet restore` and `dotnet run`
3. Test functionality, particularly anything to do with products.
4. Check the DB and make sure quantity is a field in product
